Symfony/CS/Config/InvalidFixerConfigurationException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Config;
+
+class InvalidFixerConfigurationException extends \Exception
```

Comment:
maybe a better parent class would be `\UnexpectedValueException` or `InvalidArgumentException`, any thoughts?